### PR TITLE
REVIEW: XMPP Newsletter September 2022

### PR DESCRIPTION
**Pull request by Echolon**
## This is the draft for the next release covering September 2022.

Final release is planned for 5th of October 2022.

You can make a simple comment or create a merge request to make things easy going for review. You don't need any special technical knowledge to contribute. If you need help, please ask for help in the Communication-Team MUC: [commteam@muc.xmpp.org](mailto:commteam@muc.xmpp.org)

[You can use our online Pad](https://yopad.eu/p/xmpp-newsletter-365days)! Please be tidy and write something instead of dropping bare HTTP links pray

Just paste your prepared text in this thread below or in our XMPP channel (xmpp:[commteam@muc.xmpp.org](mailto:commteam@muc.xmpp.org)?join)

Please do not just drop bare links - rather try to make a short and decent formulated description (<5 lines) what is the main outcome of your source. You do not need to write in a hard, formal tone - be passionate why this motivates you or what you work on!

Status:

XMPP Newsletter Timeline

* [x] Collection of news
* [ ] Review
* [ ] Blog post + Tiny Newsletter
* [ ] Social media
* [ ] Translations
* [ ] Archived ([Use pandoc!](https://pandoc.org/))

### CODE REVIEW

File: content/blog/2022-10-05-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+
+Kaidan, a user-friendly and modern chat app for every device, will receive a grant by NLnet for [adding encrypted audio and video calls](https://www.kaidan.im/2022/09/03/audio-video-calls/). Congratulations!
+
+### Servers
+
+Jackal [v0.62.0](https://github.com/ortuman/jackal/releases/tag/v0.62.0) is out bringing [XEP-0313](https://xmpp.org/extensions/xep-0313.html) Message Archive Management support and several enhancements.
+
+Metronome IM 4.0 [has been released](https://metronome.im/changelogs/4_0). This version introduces Lua 5.3 support, a revamped shared API for group chats stanza archiving, MIX Core/PAM experimental support, Message Moderation support and support for containerization.
+
+[Profanity 0.13.0 has been released](https://github.com/profanity-im/profanity/releases/tag/0.13.0).
```
Comment (wurstsalat3000) [inline content/blog/2022-10-05-newsletter.md:97]:
```suggestion
[Profanity 0.13.0](https://github.com/profanity-im/profanity/releases/tag/0.13.0) has been released.
```

File: content/blog/2022-10-05-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+Find the article in [English](https://nicoco.fr/blog/2022/09/04/slidge-first-beta/) and [French](https://linuxfr.org/users/therealnicoco/journaux/la-premiere-beta-de-slidge-passerelles-xmpp-est-dehors).
+
+Work on [XMPP ⇔ ActivityPub gateway](https://nlnet.nl/project/Libervia/) (supported by a [NLnel](https://nlnet.nl/) funding) has been achieved. The gateway supports features such as following/followers, favourite/like (or ["noticed" in XMPP terms](https://xmpp.org/extensions/xep-0470.html#noticed-foreword), reactions (compatible with [Pleroma](https://pleroma.social/)), mentions, sharing/reboost and lastly events (compatible with [Mobilizon](https://mobilizon.org)). It has been tested with [Mastodon](https://joinmastodon.org), Pleroma and Mobilizon. It's still considered alpha quality, [a blog post should follow soon to explain in more details](https://www.goffi.org).
+
+Ignite Realtime Community:
+  - [REST API Openfire plugin 1.10.0 released](https://discourse.igniterealtime.org/t/rest-api-openfire-plugin-1-10-0-released/92108)!
```
Comment (wurstsalat3000) [inline content/blog/2022-10-05-newsletter.md:109]:
```suggestion
  - REST API Openfire plugin 1.10.0 [has been released](https://discourse.igniterealtime.org/t/rest-api-openfire-plugin-1-10-0-released/92108)
```

File: content/blog/2022-10-05-newsletter.md [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+  - [New Openfire plugin: Push Server](https://discourse.igniterealtime.org/t/new-openfire-plugin-push-server/92069)!
```
Comment (wurstsalat3000) [inline content/blog/2022-10-05-newsletter.md:110]:
```suggestion
  - New Openfire plugin: [Push Server](https://discourse.igniterealtime.org/t/new-openfire-plugin-push-server/92069)
```

File: content/blog/2022-10-05-newsletter.md [hunk 4]
```diff
@@ -0,0 +1,195 @@
...
+- [Berlin XMPP Meetup (remote)](https://mov.im/?node/pubsub.movim.eu/berlin-xmpp-meetup): monthly meeting of XMPP enthusiasts in Berlin, every 2nd Wednesday of the month
+
+## Articles
+
+Congrats to the Monal developers for the new [NLnet Funding](https://monal-im.org/post/00003-nlnet-funding/) that will cover privacy enhancements for the push servers, an explorative audio and video calls implementation, UI updates for MUC and Contact views, and SASL updates.
+
+Paul Schaub wrote a [blog post](https://blog.jabberhead.tk/2022/09/14/using-pushdown-automata-to-verify-packet-sequences/) focused on the technical details of how to check the validity of nested packet sequences using pushdown automata. This technique will be used in a partial rework of PGPainless to validate OpenPGP messages. Earlier this month another post on ["Creating a Web-of-Trust Implementation: Accessing Certificate Stores"](https://blog.jabberhead.tk/2022/09/01/creating-a-web-of-trust-implementation-accessing-certificate-stores/) has been published, too.
+
+![PGPainless to validate OpenPGP messages](/images/newsletter/2022-10-01/TigaseWebpage.png "PGPainless to validate OpenPGP messages")
```
Comment (wurstsalat3000) [inline content/blog/2022-10-05-newsletter.md:64]:
```suggestion
![PGPainless to validate OpenPGP messages](/images/newsletter/2022-10-01/OpenPGPMessage.png "PGPainless to validate OpenPGP messages")
```

**Comment by Echolon** [comment]
## Editorial closing of the XMPP Newsletter September 2022 release

- Many thanks to everyone!!! This release to **Adrien Bourmault (neox), alkino, anubis, Benoît Sibaud, emus, goffi, Jeybe, Licaon_Kter, martin, MattJ, MSavoritias (fae,ve), nicfab, Ppjet6, vanitasvitae, wurstsalat, Zash** for their help in creation, review and translation!

- :heart: :fireworks:  Next release is next month on **5th of November 2022**!

Have a good time until then and see you soon!

**Pull request merged**